src/read_preference.ts

```diff
@@ -156,7 +156,9 @@ export class ReadPreference {
     }
 
     if (typeof readPreference === 'string') {
-      return new ReadPreference(readPreference as ReadPreferenceMode, readPreferenceTags);
+      return new ReadPreference(readPreference as ReadPreferenceMode, readPreferenceTags, {
+        hedge: options.hedge
```

Comment:
It wasn't mentioned in this ticket but I would imagine we also have a gap here if options has a `maxStalenessSeconds` defined on it. It's used in the other contstruction on L165, so I think it should be here too.

Reply:
Yup, I forgot to mention that in the description.  I wasn't sure if I should include that as a part of my PR but that's an easy change to make as well, so I'll add it!